lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -350,10 +349,29 @@ class CategoryRenderer {
       percentageEl.title = Util.i18n.strings.errorLabel;
     }
 
+    if (this.hasApplicableAudits(category)) {
+      wrapper.classList.add(`lh-gauge__wrapper--${Util.calculateRating(category.score)}`);
+    } else {
+      // Render a gray dash for entirely non-applicable categories.
+      wrapper.classList.add(`lh-gauge__wrapper--notapplicable`);
+      percentageEl.textContent = '-';
+      percentageEl.style.color = '#757575';
```

Comment:
this should be applied to the class in the css.

Reply:
ty fixed